content/general/concepts/what-is-functional-programming/what-is-functional-programming.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'Functional Progtamming'
+Description: 'Functional Programming (FP) is a declarative software development paradigm which encourages constructing programs by composing pure functions and evaluating expressions instead of statements. The functions are isolated and independent of the state of the application. In Functional Programming, dependencies such as variables or objects needed for a function, are always declared explicitly by passing them into the function as arguments. Any operation inside the function is therefore binded to the arguments passed, instead of depending on global objects or variables. This makes the function easier to test and debug, and allows for more predictability, reusability and stability when altering and refactoring code.'
```

Comment:
```suggestion
Description: 'Encourages programming with pure functions and evaluating expressions instead of statements.'
```

For Docs entries, per the [entry template](https://raw.githubusercontent.com/Codecademy/docs/main/documentation/entry-template.md), can we keep the length of the meta-`Description` within (ideally) 150 characters

Reply:
I see, makes sense! Actually I used the existing entry on OOP as some guidance to write this one, and the description was quite lengthy there, but should have checked the template better!